backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5565660_sys_gh9838_AddSeqNoInAD_Language.sql

```diff
@@ -0,0 +1,60 @@
+;
+
+-- 2020-08-25T09:13:06.016Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+/* DDL */  select update_Column_Translation_From_AD_Element(566) 
+;
+
+-- 2020-08-25T09:13:32.511Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+UPDATE AD_Column SET DefaultValue='0',Updated=TO_TIMESTAMP('2020-08-25 12:13:32','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Column_ID=571120
+;
+
+-- 2020-08-25T09:13:33.661Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+/* DDL */ SELECT public.db_alter_table('AD_Language','ALTER TABLE public.AD_Language ADD COLUMN SeqNo NUMERIC(10) DEFAULT 0 NOT NULL')
+;
+
+-- 2020-08-25T09:13:38.256Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+UPDATE AD_Column SET DefaultValue='@SQL=SELECT COALESCE(MAX(SeqNo),0)+10 AS DefaultValue FROM AD_Language',Updated=TO_TIMESTAMP('2020-08-25 12:13:38','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Column_ID=571120
+;
+
+-- 2020-08-25T09:13:39.973Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO t_alter_column values('ad_language','SeqNo','NUMERIC(10)',null,null)
+;
+
+-- 2020-08-25T09:14:14.893Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Field (AD_Tab_ID,IsDisplayed,DisplayLength,SortNo,IsSameLine,IsHeading,IsFieldOnly,IsEncrypted,AD_Client_ID,IsActive,Created,CreatedBy,IsReadOnly,ColumnDisplayLength,IncludedTabHeight,Updated,UpdatedBy,Help,AD_Field_ID,IsDisplayedGrid,SeqNo,SeqNoGrid,SpanX,SpanY,AD_Column_ID,Name,Description,EntityType,AD_Org_ID) VALUES (112,'Y',0,0,'N','N','N','N',0,'Y',TO_TIMESTAMP('2020-08-25 12:14:14','YYYY-MM-DD HH24:MI:SS'),100,'N',0,0,TO_TIMESTAMP('2020-08-25 12:14:14','YYYY-MM-DD HH24:MI:SS'),100,'"Reihenfolge" bestimmt die Reihenfolge der Einträge',616356,'Y',150,150,1,1,571120,'Reihenfolge','Zur Bestimmung der Reihenfolge der Einträge; die kleinste Zahl kommt zuerst','D',0)
+;
+
+-- 2020-08-25T09:14:14.895Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Field_Trl (AD_Language,AD_Field_ID, Help,Name,Description, IsTranslated,AD_Client_ID,AD_Org_ID,Created,Createdby,Updated,UpdatedBy) SELECT l.AD_Language, t.AD_Field_ID, t.Help,t.Name,t.Description, 'N',t.AD_Client_ID,t.AD_Org_ID,t.Created,t.Createdby,t.Updated,t.UpdatedBy FROM AD_Language l, AD_Field t WHERE l.IsActive='Y'AND (l.IsSystemLanguage='Y' AND l.IsBaseLanguage='N') AND t.AD_Field_ID=616356 AND NOT EXISTS (SELECT 1 FROM AD_Field_Trl tt WHERE tt.AD_Language=l.AD_Language AND tt.AD_Field_ID=t.AD_Field_ID)
+;
+
+-- 2020-08-25T09:14:14.899Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+/* DDL */  select update_FieldTranslation_From_AD_Name_Element(566) 
+;
+
+-- 2020-08-25T09:14:14.915Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+DELETE FROM AD_Element_Link WHERE AD_Field_ID=616356
```

Comment:
after you went this far, please also add the seqno field to the UI...so the user can change it

Reply:
@metas-ts but it is the ui; not sure what's with that delete; checking